opentok/opentok.py

```diff
@@ -1365,3 +1367,66 @@ def __init__(
             timeout=timeout,
             app_version=app_version
         )
+
+    
+
+
+    def mute(self, session_id: str, stream_id: str= "", options: dict = {}) -> requests.Response:
+        """
+        Use this method so the moderator can mute all streams or a specific stream for OpenTok.
+        Please note that a client is able to unmute themselves.
+        This function stays in the OpenTok class and inherits from the Client class.
+
+        :param session_id gets the session id from another function called get_session()
+
+        :param stream_id gets the stream id from another function called get_stream(). Note
+        that this variable is set to an empty string in the function definition as a specific
+        stream may not be chosen.
+   
+        """
+
+        session_id = self.get_session()
+        stream_id = self.get_stream()
+        jwt = self._create_jwt_auth_header()
+
+        if not stream_id:
+            response = requests.post(
+            f'https://api.opentok.com/v2/project/{os.getenv("API_SECRET")}/{session_id}/stream/mute',
+            data={'excludedStream': streamId},
+            headers={'Content-Type':'application/json', 
+                    'X-OPENTOK-AUTH': jwt})
+        else:
+            response = requests.post(
+            f'https://api.opentok.com/v2/project/{os.getenv("API_SECRET")}/{session_id}/stream/{stream_id}/mute',
+            headers={'X-OPENTOK-AUTH': jwt})
```

Comment:
Related to above, by `self.get_headers()` will return an object with all the needed headers for a request.

https://github.com/opentok/Opentok-Python-SDK/blob/d7607c1e4ff25a20848c4d6143e76807ff51ad95/opentok/opentok.py#L440

Reply:
Replaced the headers with the `self.get_headers()` function.